.circleci/integration-tests/Dockerfile

```diff
@@ -14,6 +14,18 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 RUN unzip awscliv2.zip
 RUN ./aws/install
 
+# install eksctl
+RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+RUN mv /tmp/eksctl /usr/local/bin
+RUN eksctl version
```

Comment:
Trying to understand the objective of having this command. Can you please explain the purpose? If it not necessary, we can remove this line as it will add an unnecessary layer in the docker image.

Reply:
Added as single line